src/main/kotlin/de/amplimind/codingchallenge/service/UserService.kt

```diff
@@ -407,4 +409,12 @@ class UserService(
             this.resetPasswordTokenStorage.addToken(changePasswordRequestDTO.token)
         }
     }
+
+    /**
+     * checks if the current user is an admin
+     * @return if the current user is an admin
+     */
+    fun isLoggedInUserAdmin(): IsAdminDTO {
```

Comment:
Methoden welche mit is / has beginnen sind eig. immer ein Indiz dafür dass ein Boolean returned wird

Reply:
An sich ist das DTO auch nur der Wrapper von nem Boolean